tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,293 @@
-
-##############################################################################
-# Define peak-to-peak rejection parameters for gradiometers, magnetometers
-# and EOG:
-
-reject = dict(grad=4000e-13, mag=4e-12, eog=150e-6)
-
-##############################################################################
-# Read epochs:
-
-epochs = mne.Epochs(raw, events, event_id, tmin, tmax, proj=True,
-                    picks=('meg', 'eeg', 'eog'),  # or could pass "picks" here
-                    baseline=baseline, preload=False, reject=reject)
-print(epochs)
-
-##############################################################################
-# Get single epochs for one condition:
-
-epochs_data = epochs['aud_l'].get_data()
-print(epochs_data.shape)
-
-##############################################################################
-# epochs_data is a 3D array of dimension (55 epochs, 365 channels, 106 time
-# instants).
+# +----------+----------------------------------------------------------+
+# | Event ID | Condition                                                |
+# +==========+==========================================================+
+# | 1        | auditory stimulus (tone) to the left ear                 |
+# +----------+----------------------------------------------------------+
+# | 2        | auditory stimulus (tone) to the right ear                |
+# +----------+----------------------------------------------------------+
+# | 3        | visual stimulus (checkerboard) to the left visual field  |
+# +----------+----------------------------------------------------------+
+# | 4        | visual stimulus (checkerboard) to the right visual field |
+# +----------+----------------------------------------------------------+
+# | 5        | smiley face (catch trial)                                |
+# +----------+----------------------------------------------------------+
+# | 32       | subject button press                                     |
+# +----------+----------------------------------------------------------+
+
+event_dict = {'auditory/left': 1, 'auditory/right': 2, 'visual/left': 3,
+              'visual/right': 4, 'smiley': 5, 'buttonpress': 32}
+
+###############################################################################
+# Event dictionaries like this one are used when extracting epochs from
+# continuous data; the ``/`` character in the dictionary keys allows pooling
+# across conditions by requesting partial condition descriptors (i.e.,
+# requesting ``'auditory'`` will select all epochs with Event IDs 1 and 2;
+# requesting ``'left'`` will select all epochs with Event IDs 1 and 3). An
+# example of this is shown in the next section. There is also a convenient
+# :func:`~mne.viz.plot_events` function for visualizing the distribution of
+# events across the duration of the recording (to make sure event detection
+# worked as expected):
+
+fig = mne.viz.plot_events(events, event_id=event_dict, sfreq=raw.info['sfreq'])
```

Comment:
I think it might be worth introducing (very briefly!) that the metadata is stored in `raw.info` and showing that it contains information like sampling frequency, channel names and lowpass/highpass. Later in the tutorial, when you construct an epochs object or evoked object, you can show that in fact the info object is passed on

Reply:
+1 for raw.info, could be mentioned up around L59?